canonicalwebteam/blog/blog_api.py

```diff
@@ -159,7 +159,16 @@ def _transform_article(self, article):
                 + article["image"]["source_url"]
                 + '" loading="lazy">'
             )
-
+        if "yoast_head" in article:
+            # extracting and parsing header metadata
+            yoast_head = article.get('yoast_head', '')
+            soup = BeautifulSoup(yoast_head, 'html.parser')
+            
+            meta_description_tag = soup.find('meta', attrs={'name': 'description'})
+
+            if meta_description_tag:
```

Comment:
We could use [exert](https://github.com/canonical/canonicalwebteam.blog/blob/330eda30eceeefcb16d34b00787a0d7e13609a84/canonicalwebteam/blog/blog_api.py#L100) as a fall back here, if 'yoast' is not present